bundles/amazon/ack/elasticache/bundle/config/00-schema.yml

```diff
@@ -0,0 +1,63 @@
+#@data/values-schema
+---
+#@schema/title "Name"
+#@schema/desc "Name of the ElastiCache instance"
+name: ""
+
+#@schema/title "Namespace"
+#@schema/desc "Namespace to deploy the kubernetes resources to"
+namespace: ""
+
+#@schema/title "CreateNamespace"
+#@schema/desc "Whether to create the namespace for the resources or not"
+createNamespace: False
+
+#@schema/title "CacheSubnetGroupName"
+#@schema/desc "Name of the cache subnet group to create the replication group in. It has to be available upfront."
```

Comment:
*nitpick*
```suggestion
#@schema/desc "Name of the cache subnet group to create the replication group in. It must be pre-created."
```

Reply:
As non-English native speaker, I accept all kinds of suggestions like this one 🙂